File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/repository/jpa/CakeShopOperationJpaRepository.java
package com.cakk.domain.mysql.repository.jpa;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;

import com.cakk.domain.mysql.entity.shop.CakeShopOperation;

public interface CakeShopOperationJpaRepository extends JpaRepository<CakeShopOperation, Long> {

	List<CakeShopOperation> findAllByCakeShopId(Long cakeShopId);
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/repository/jpa/CakeShopOperationJpaRepository.kt
package com.cakk.domain.mysql.repository.jpa

import com.cakk.domain.mysql.entity.shop.CakeShopOperation
import org.springframework.data.jpa.repository.JpaRepository

interface CakeShopOperationJpaRepository : JpaRepository<CakeShopOperation, Long> {

	fun findAllByCakeShopId(cakeShopId: Long): List<CakeShopOperation>
}

